scripts/scratch_query/upload_to_bq.py

```diff
@@ -0,0 +1,57 @@
+"""Upload NDJSON file to BigQuery
+
+DESCRIPTION
+This CLI takes a local NDJSON file and appends it to an existing bigquery table.
+(reference: https://cloud.google.com/bigquery/docs/loading-data-local#loading_data_from_a_local_data_source and
+https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-json#loading_json_data_into_a_new_table)
+
+EXAMPLE
+$ python upload_to_bq.py ../../tests/data/valid_arrays_v1.1.3_for_bq_v1.json
```

Comment:
This example lacks values for the positional arguments `table_id` and `dataset_id`, which you define below.  I assume they're required, as is conventional for positional arguments, so my guess is this example command fails.

If that's correct, could you fix the example command?

Reply:
Great catch. I've update the example command in 31c8281